NYPLCardCreator/Classes/UserSummaryViewController.swift

```diff
@@ -0,0 +1,239 @@
+
+    self.cells = [
+      self.homeAddressCell,
+      self.fullNameCell,
+      self.emailCell,
+      self.usernameCell,
+      self.pinCell
+    ]
+    self.sectionHeaderTitles = [
+      "Home Address",
+      "Full Name",
+      "Email",
+      "Username",
+      "Pin"
+    ]
+    if (self.schoolOrWorkAddress != nil) {
+      self.cells.insert(self.altAddressCell, atIndex: 1)
+      self.sectionHeaderTitles.insert("School or Work Address", atIndex: 1)
+    }
+    
+    super.init(style: .Grouped)
+    self.tableView.estimatedRowHeight = 104
+    
+    self.navigationItem.rightBarButtonItem =
+      UIBarButtonItem(title: NSLocalizedString("Create Card", comment: "A title for a button that submits the user's information to create a library card"),
+                      style: .Plain,
+                      target: self,
+                      action: #selector(createPatron))
+
+    self.prepareTableViewCells()
+  }
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    self.title = NSLocalizedString(
+      "Review",
+      comment: "A title for a screen letting the user know they can review the information they have entered")
+    
+    let headerView = UIView(frame: CGRect(x: 0, y: 0, width: self.view!.bounds.size.width, height: 80.0))
+    let headerLabel = UILabel()
+    headerView.addSubview(headerLabel)
+    
+    headerLabel.autoPinEdgesToSuperviewEdgesWithInsets(UIEdgeInsetsMake(2, 22, 2, 22))
+    headerLabel.numberOfLines = 2
+    headerLabel.textColor = UIColor.darkGrayColor()
+    headerLabel.textAlignment = .Center
+    headerLabel.text = NSLocalizedString("Review your information before creating your library card.", comment: "Description to tell a user to either review and confirm, or go back and make changes to their information.")
+    
+    self.tableView.tableHeaderView = headerView
+    self.tableView.userInteractionEnabled = false
+  }
+  
+  private func prepareTableViewCells() {
+    for cell in self.cells {
+      cell.backgroundColor = UIColor.clearColor()
+      self.tableView.separatorStyle = .None
+      
+      if let addressViewCell = cell as? AddressCell {
+        //style as an address cell
+      }
+      
+      if let labelledTextViewCell = cell as? LabelledTextViewCell {
+        labelledTextViewCell.selectionStyle = .None
+        labelledTextViewCell.textField.allowsEditingTextAttributes = false
+      }
+    }
+  }
+  
+  // MARK: UITableViewDataSource
+  
+  override func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+    return 1
+  }
+  
+  func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+    return self.cells.count
+  }
+  
+  override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+    return self.cells[indexPath.section]
+  }
+  
+  func tableView(tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
+    return 1.0
```

Comment:
Surely this cannot be correct! Is there a reason we return a height for footers here?